util/src/main/java/com/netcracker/ConsoleHelper.java

```diff
@@ -0,0 +1,39 @@
+package com.netcracker;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class ConsoleHelper {
+    private static final BufferedReader READER = new BufferedReader(new InputStreamReader(System.in));
+
+    private ConsoleHelper(){
+        throw new RuntimeException("Don't create instance of this class");
+    }
+
+    public static void writeMessage(String message){
+        System.out.println(message);
+    }
+
+    public static String readMessage(){
+        while(true) {
+            try {
+                return READER.readLine();
+            } catch (IOException e) {
+                writeMessage("Error. Please try again.");
+            }
+        }
+    }
+
+    public static int readInt(){
+        while(true) {
+            try {
+                String one = readMessage();
+                return Integer.parseInt(one);
+            } catch (NumberFormatException e) {
+                writeMessage("Can't recognize number. Please try again.");
```

Comment:
I think, that when validation exception thrown, you better rethrow the exception to the caller and the caller will use writeMessage to inform the user about what is going on.
This particular method shouldn't know about "the User" and "Please try again." - its the obligation of the outer abstraction layer. All this method cares about is reading an input stream and parsing an integer.